src/components/calculator.js

```diff
@@ -1,6 +1,7 @@
 import React, { useState } from 'react';
 import './style.css';
 import calculate from '../logic/calculate';
+import QuoteDisplay from './fetchQuote';
```

Comment:
- [ ] The quote component should be imported into the App component, not the Calculator component. Kindly fix that.